hierarchicalforecast/methods.py

```diff
@@ -419,6 +418,8 @@ def min_trace(S: np.ndarray,
               y_insample: np.ndarray,
               y_hat_insample: np.ndarray,
               method: str,
+              idx_bottom: List[int] = None,
```

Comment:
Would it be possible to infer the idx_bottom indexes from the S_[a+b, b] matrix?
something like  `idx_bottom = n.arange(a+b)[a:]`

If we commit to ordered numpy arrays, things might simplify.

Additionally the PERMBU method is operating under the assumption of hierarchically ordered numpy arrays.

Reply:
I think we could test the equality of the following arrays: `y_[a] = A y_[b]`. To test for hierarchically ordered arrays.
Assuming testable properties in the numpy arrays would allow us to simplify the code and make it more efficient.

Reply:
> I think we could test the equality of the following arrays: y_[a] = A y_[b] for hierarchically order property of the input arrays.

And then you will include a new argument, `y_[b]`?

Reply:
I'm not changing the `idx_bottom` since it is used in other methods. If you want to test your idea feel free to open an issue and make a PR. The `idx_bottom` argument is compatible with the previous structure.